CMakeLists.txt

```diff
@@ -583,8 +583,11 @@ add_subdirectory(main)
 add_subdirectory(nanofi)
 add_dependencies(nanofi minifiexe)
 
-add_subdirectory(encrypt-config)
-add_dependencies(encrypt-config minifi)
+option(DISABLE_ENCRYPT_CONFIG "Disables build of encrypt-config binary." OFF)
+if (NOT DISABLE_ENCRYPT_CONFIG)
```

Comment:
There was a discussion about double negatives in the ConsumeJournald PR. It should apply here, too. https://github.com/apache/nifi-minifi-cpp/pull/1044#discussion_r622005451

Reply:
Thanks, updated in d6f56f9a2a7808d459cb72afc7da5f2f091fd710